src/Tribe/Tickets_Handler.php

```diff
@@ -358,77 +359,73 @@ public function get_object_connections( $object ) {
 			return $connections;
 		}
 
-		$provider_index = array(
-			'rsvp' => 'Tribe__Tickets__RSVP',
-			'tpp'  => 'Tribe__Tickets__Commerce__PayPal__Main',
-			'woo'  => 'Tribe__Tickets_Plus__Commerce__WooCommerce__Main',
-			'edd'  => 'Tribe__Tickets_Plus__Commerce__EDD__Main',
-		);
+		/** @var Tribe__Tickets__Status__Manager $status */
+		$status = tribe( 'tickets.status' );
 
-		$relationships = array(
-			'event' => array(
+		$relationships = [
+			'event'      => [
 				// RSVP
-				'_tribe_rsvp_event' => 'rsvp',
-				'_tribe_rsvp_for_event' => 'rsvp',
+				'_tribe_rsvp_event'          => 'rsvp',
+				'_tribe_rsvp_for_event'      => 'rsvp',
 
 				// PayPal tickets
-				'_tribe_tpp_event' => 'tpp',
-				'_tribe_tpp_for_event' => 'tpp',
+				'_tribe_tpp_event'           => 'tpp',
+				'_tribe_tpp_for_event'       => 'tpp',
 
 				// EDD
-				'_tribe_eddticket_event' => 'edd',
+				'_tribe_eddticket_event'     => 'edd',
 				'_tribe_eddticket_for_event' => 'edd',
 
 				// Woo
-				'_tribe_wooticket_event' => 'woo',
+				'_tribe_wooticket_event'     => 'woo',
 				'_tribe_wooticket_for_event' => 'woo',
-			),
-			'product' => array(
+			],
+			'product'    => [
 				// RSVP
-				'_tribe_rsvp_product' => 'rsvp',
+				'_tribe_rsvp_product'      => 'rsvp',
 
 				// PayPal tickets
-				'_tribe_tpp_product' => 'tpp',
+				'_tribe_tpp_product'       => 'tpp',
 
 				// EDD
 				'_tribe_eddticket_product' => 'edd',
 
 				// Woo
 				'_tribe_wooticket_product' => 'woo',
-			),
-			'order' => array(
+			],
+			'order'      => [
 				// RSVP
-				'_tribe_rsvp_order' => 'rsvp',
+				'_tribe_rsvp_order'      => 'rsvp',
 
 				// PayPal tickets
-				'_tribe_tpp_order' => 'tpp',
+				'_tribe_tpp_order'       => 'tpp',
 
 				// EDD
 				'_tribe_eddticket_order' => 'edd',
 
 				// Woo
 				'_tribe_wooticket_order' => 'woo',
 
-			),
-			'order_item' => array(
+			],
+			'order_item' => [
 				// PayPal tickets
-				'_tribe_tpp_order' => 'tpp',
+				'_tribe_tpp_order'            => 'tpp',
 
 				// Woo
 				'_tribe_wooticket_order_item' => 'woo',
-			),
-		);
+			],
+		];
 
```

Comment:
```suggestion

		/**
		 * Allow filtering the relationships so providers can add their own strings.
		 *
		 * @since TBD
		 *
		 * @param array $relationships List of relationship information for providers.
		 */
		 $relationships = apply_filters( 'tribe_tickets_handler_relationships', $relationships );

```